Signal/src/UIColor+OWS.m

```diff
@@ -107,4 +107,21 @@ + (UIColor *)backgroundColorForContact:(NSString *)contactIdentifier
     return [colors objectAtIndex:(choose % [colors count])];
 }
 
++ (UIColor *)colorWithRGBHex:(unsigned long)value
+{
+    CGFloat red = ((value >> 16) & 0xff) / 255.f;
+    CGFloat green = ((value >> 8) & 0xff) / 255.f;
+    CGFloat blue = ((value >> 0) & 0xff) / 255.f;
+    return [UIColor colorWithRed:red green:green blue:blue alpha:1.f];
+}
+
++ (UIColor *)colorWithARGBHex:(unsigned long)value
```

Comment:
> ARGB is the most common ordering for this kind of hex representation for colors, right?

I certainly didn't think so, else I wouldn't have brought it up.

See also: https://en.wikipedia.org/wiki/RGBA_color_space